src/styles.js

```diff
@@ -53,6 +53,9 @@ const useStyles = makeStyles((theme) => ({
     '.containerWhite': {
       backgroundColor: theme.palette.background.default,
     },
+    '.containerTeal': {
+      backgroundColor: theme.palette.secondary.dark,
+    },
     '.containerWorld': {
```

Comment:
The containerTeal class is already present in styles.js.This is a duplicate class. Otherwise, the code works fine.

Reply:
fixed